BrainPortal/app/views/groups/show.html.erb

```diff
@@ -91,14 +91,10 @@
 
   <% end %>
 
-  <% # Access Profile List %>
-  <% if current_user.has_role?(:admin_user) && @group.access_profiles.count > 0 %>
-    <%= show_table(@group, :header => 'Access profiles') do |t| %>
-      <% t.cell("Access Profiles", :no_header => true) do %>
-        <%= array_to_table(@group.access_profiles.order(:name).all, :table_class => "simple", :cols => 4) do |access_profile| %>
-          <%= access_profile_label(access_profile, :with_link => true) %>
-        <% end %>
-      <% end %>
+  <% if current_user.has_role?(:admin_user) %>
+    <% group_access_profiles = (@group.access_profiles.map { |ap| access_profile_label(ap) }.join("&nbsp&nbsp").html_safe.presence) || "(None)" %>
+    <%= show_table(@group, :header => 'Access Profiles') do |t| %>
+      <% t.cell("Access Profiles") { group_access_profiles } %>
```

Comment:
I'll provide screenshots in the main discussion: we should adjust the numbers of columns in array_to_table.